en/lessons/specifics/ecto_basics.md

```diff
@@ -0,0 +1,194 @@
+---
+version: 1.3.0
```

Comment:
@jeremyjung / @notactuallypagemcconnell since this is an entirely new lesson (even though a take on the original), what do you think about starting over at `1.0.0`?

cc @gemantzu / @brain-geek / @nscyclone for comment from the translator perspective.

Reply:
@gemantzu I believe the plan is to move this into the ecto subsection as the basics.

@notactuallypagemcconnell / @SophieDeBenedetto correct me if I'm mistaken.